app/views/books/edit.html.erb

```diff
@@ -1,6 +1,15 @@
-<h1>Editing Book</h1>
+<div style="margin-top:50px;" class="container-fluid col-md-8 col-md-offset-2">
+  <div class="panel panel-primary">
 
-<%= render 'form', book: @book %>
+    <div class="panel-heading">
+      <div class="panel-title">Edit book</div>
```

Comment:
Good call out. I'm taking some time to read about best practices and this has been one of the points I've come across.